packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -61,7 +61,8 @@ export default class TableResize implements EditorPlugin {
         }
 
         this.ensureTableRects();
-        const { pageX: x, pageY: y } = e;
+        const x = e.pageX - window.scrollX;
```

Comment:
This is a good finding. I'm afraid each place where e.pageX/Y is used will have the same issue.

And here instead of using the global window, we need to retrieve window object from editor instead, because the global window may not be the window that editor is in.

Reply:
Ok, but then I'm afraid I don't know the code well enough to fix this in all the necessary places.

Reply:
We can fix other place separately. Please just change the global window here to:
```ts
  this.editor.getDocument().defaultView
```

Thanks.